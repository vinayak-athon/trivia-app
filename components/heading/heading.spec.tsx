import * as React from "react";
import { render } from "@Test/utils";
import { Heading } from "@Components";

describe("Components", () => {
  describe("Heading", () => {
    it("should render given text", () => {
      const { getAllByText } = render(<Heading text="This is a heading" />);

      expect(getAllByText("This is a heading").length).toEqual(1);
    });

    it("should match snapshot", () => {
      const { container } = render(<Heading text="Toxic" />);

      expect(container).toMatchSnapshot();
    });
  });
});
