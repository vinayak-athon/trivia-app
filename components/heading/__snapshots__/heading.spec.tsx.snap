// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Heading should match snapshot 1`] = `
.c0 {
  font-family: Arial,Helvetica,sans-serif;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  position: relative;
  z-index: 1;
  text-align: center;
}

.c0 h3 {
  color: #000000;
  font-size: 20px;
  font-weight: 800;
}

<div>
  <div
    class="c0"
  >
    <h3>
      Toxic
    </h3>
  </div>
</div>
`;
